File: 백준/Silver/9095. 1， 2， 3 더하기/1， 2， 3 더하기.kt
fun main()
{
    val br=System.`in`.bufferedReader()
    val bw=System.`out`.bufferedWriter()
    var arr= IntArray(12)
    arr[0]=1
    for(i in 1..11)
    {
        for(j in -3..-1)
        {
            if(i+j>=0)arr[i]+=arr[i+j]
        }
    }
    val T=br.readLine().toInt()
    for (i in 1..T)
    {
        val n=br.readLine().toInt()
        bw.write("${arr[n]}\n")
    }
    bw.close()
}

File: 백준/Silver/9095. 1， 2， 3 더하기/1， 2， 3 더하기.py
arr=[0 for _ in range(12)]
arr[0]=1
for i in range(1,12):
    for j in range(-3,0):
        if i+j>=0:
            arr[i]+=arr[i+j]
T=int(input())
for _ in range(T):
    n=int(input())
    print(arr[n])